pasta

```diff
@@ -43,7 +43,7 @@ from bin.pasta_patch_descriptions import patch_descriptions
 from bin.pasta_upstream_duration import upstream_duration
 from bin.pasta_upstream_history import pasta_upstream_history
 from bin.pasta_web import web
-from bin.pasta_show_maintainers import get_maintainers
+from bin.pasta_show_maintainers import show_maintainers
```

Comment:
No need to change this line, if you use the final name (show_maintainers) in the first commit.